Lib/test/test_codecs.py

```diff
@@ -3124,105 +3199,99 @@ def set_codec(self, encode, decode):
     @unittest.expectedFailure
     def test_raise_subclass_with_weakref_support(self):
-        msg = "This should be wrapped"
+        msg = "This should be noted"
         class MyRuntimeError(RuntimeError):
             pass
-        self.check_wrapped(MyRuntimeError(msg), msg, MyRuntimeError)
+        self.check_note(MyRuntimeError(msg), msg, MyRuntimeError)
 
-    def check_not_wrapped(self, obj_to_raise, msg):
-        def raise_obj(*args, **kwds):
-            raise obj_to_raise
-        self.set_codec(raise_obj, raise_obj)
-        with self.assertRaisesRegex(RuntimeError, msg):
-            "str input".encode(self.codec_name)
-        with self.assertRaisesRegex(RuntimeError, msg):
-            codecs.encode("str input", self.codec_name)
-        with self.assertRaisesRegex(RuntimeError, msg):
-            b"bytes input".decode(self.codec_name)
-        with self.assertRaisesRegex(RuntimeError, msg):
-            codecs.decode(b"bytes input", self.codec_name)
-
-    @unittest.expectedFailureIfWindows("TODO: RUSTPYTHON")
-    def test_init_override_is_not_wrapped(self):
+    # TODO: RUSTPYTHON
+    @unittest.expectedFailure
+    def test_init_override(self):
         class CustomInit(RuntimeError):
             def __init__(self):
                 pass
-        self.check_not_wrapped(CustomInit, "")
-
-    @unittest.expectedFailureIfWindows("TODO: RUSTPYTHON")
-    def test_new_override_is_not_wrapped(self):
+        self.check_note(CustomInit, "")
+        
+    # TODO: RUSTPYTHON
+    @unittest.expectedFailure  
+    def test_new_override(self):
         class CustomNew(RuntimeError):
             def __new__(cls):
                 return super().__new__(cls)
-        self.check_not_wrapped(CustomNew, "")
+        self.check_note(CustomNew, "")
 
-    @unittest.expectedFailureIfWindows("TODO: RUSTPYTHON")
-    def test_instance_attribute_is_not_wrapped(self):
-        msg = "This should NOT be wrapped"
+    # TODO: RUSTPYTHON
+    @unittest.expectedFailure
+    def test_instance_attribute(self):
+        msg = "This should be noted"
         exc = RuntimeError(msg)
         exc.attr = 1
-        self.check_not_wrapped(exc, "^{}$".format(msg))
+        self.check_note(exc, "^{}$".format(msg))
 
-    @unittest.expectedFailureIfWindows("TODO: RUSTPYTHON")
-    def test_non_str_arg_is_not_wrapped(self):
-        self.check_not_wrapped(RuntimeError(1), "1")
+    # TODO: RUSTPYTHON
+    @unittest.expectedFailure
+    def test_non_str_arg(self):
+        self.check_note(RuntimeError(1), "1")
 
-    @unittest.expectedFailureIfWindows("TODO: RUSTPYTHON")
-    def test_multiple_args_is_not_wrapped(self):
+    # TODO: RUSTPYTHON
+    @unittest.expectedFailure    
+    def test_multiple_args(self):
         msg_re = r"^\('a', 'b', 'c'\)$"
-        self.check_not_wrapped(RuntimeError('a', 'b', 'c'), msg_re)
+        self.check_note(RuntimeError('a', 'b', 'c'), msg_re)
 
-    @unittest.expectedFailureIfWindows("TODO: RUSTPYTHON")
```

Comment:
This test seems only fails on Windows. Keeping this line instead of naive `expectedFailure` will resolve the CI failure.